### DISCUSSION TITLE
Limit the number of concurrent matches to 5 or something, so that people can focus on giving it a real shot with those people.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#85] [Ideas & Features]
The paradox of choice means that having more options often does not make people happier or lead to better outcomes. For this reason, having a limited supply of matches to pick from at a given time could make users (particularly women) more responsive to the ones they do have. Then if they want to go back to swiping, then they would have to unmatch someone explicitly.

**Comment by Oshyan** [comment]
I think this could be a useful limit. But I would definitely want to still be able to look at the "Like queue", i.e. the people who have liked me, and be able to e.g. "swap" someone out for someone new. Maybe further to that, have it be kind of like a 3-part match flow. It starts with open matching, and you can always Like or view who Likes you, unlimited queue. Then if you want to mutual Match with someone and chat, you can only have 5 active at a time. If you swap someone out, maybe they go into a "Previous Matches" area where you can't chat or see messages from them (but maybe can see that there is a message waiting *from* them, if they have kept you in *their* messaging queue), but you can still see all the people you've matched with and not explicitly "unmatched" (i.e. blocked?). And perhaps you could still re-match with them later if desired. A good example of why one might want to do that is let's say someone in your messaging queue (limit 5) went quiet for a few days and there's another match you're really interested in messaging. You swap them out, only to find that the really interesting match is not so interesting to actually talk to, and meanwhile the other person who is now in your "Previous Matches" queue has a waiting message and you want to read it. To avoid people swapping others in and out quickly so that they can sort of get around the limit, you could make it only 1 swap per profile in a 24hr period or something.

**Comment by barvian** [comment]
The lead scientist (I believe) at Match has  recommended people limit concurrent matches to 9. Here's [one article](https://www.vox.com/2019/2/7/18210998/tinder-algorithm-swiping-tips-dating-app-science)  where she's quoted.

**Reply by WilliamDormer** [reply]
I can't see the 9 match limit in this article though..

**Reply by barvian** [reply]
"She recommended that anyone using a dating app should stop swiping as soon as they have nine matches — the highest number of choices our brain is equipped to deal with at one time."
